### ISSUE TITLE
Split up "html" label to multiple labels

### ISSUE THREAD

**Issue by zcorpan** [web-platform-tests/wpt-pr-bot#129]
HTML is a big spec, but wpt-pr-bot only sets `html` label for things changed under `html/`.

I think it would be worthwhile to have separate labels. The simplest implementation is probably to have subdirectories get their own labels, but it might not be the most useful one. In particular, the `semantics` subdirectory still has separate features like `forms` and `embedded-content`.

The whatwg/html repo has [44 "topic:" labels](https://github.com/whatwg/html/labels?utf8=%E2%9C%93&q=topic). Although a one-to-one mapping to them doesn't seem necessary, at least those labels were created separately because that was thought to be useful.

As another consideration, some features are defined across several sections of the spec.

Maybe `META.yml` could be used to specify the appropriate labels?

**Comment by foolip** [comment]
Spelling out the labels in `META.yml` instead of having wpt-pr-bot maintain have that logic would be an improvement IMHO. @stephenmcgruer WDYT?

**Comment by stephenmcgruer** [comment]
I agree that it sounds like a reasonable enhancement, but I don't see a high priority here. Have we heard any concerns about the current labeling, or any requests for it to be improved?

Thinking about even the use of labeling at all, I would hazard a guess that maybe people interested in a particular spec might filter PRs by that label to get an overview of changes - but I don't concretely know of anyone doing that (and whether this proposed change would be useful to them).

**Comment by foolip** [comment]
The labels are used for filtering, unfortunately it's not possible to customize notifications (email) based on labels. Most prominent is probably in the "ongoing work" links in WHATWG specs, see https://dom.spec.whatwg.org/ for example.

The wg-* labels are a bit of a nuisance IMHO. https://github.com/web-platform-tests/wpt/issues/10707 is an issue about it not working, but to fix it requires doing something with https://github.com/tobie/specref. Just enumerating the labels that should apply to a directory would be easier.

Then there's the kind of request in this issue: more granular labels.
